package.json

```
@@ -125,6 +125,16 @@
                 "category": "DeepScan",
                 "command": "deepscan.clearProject",
                 "title": "Clear Project Problems"
+            },
+            {
+                "category": "DeepScan",
+                "command": "deepscan.setToken",
+                "title": "Configure DeepScan Access Token For VS Code"
```

커맨드 이름은 최대한 간단하게 하는 것이 좋을 듯 합니다.
DeepScan은 prefix로 붙으니 이름에 다시 넣을 필요 없을 듯 하고요.

```suggestion
                "title": "Configure Access Token"
```